sfepy/terms/terms_flexo.py

```diff
@@ -0,0 +1,190 @@
+"""
+Flexoelectricity related terms.
+"""
+import numpy as nm
+
+from sfepy.mechanics.tensors import dim2sym
+from sfepy.terms.terms_multilinear import ETermBase
+
+def make_grad2strain(dim):
```

Comment:
what happens if `dim` is not 1, 2, or 3?

Reply:
Thanks, fixed!